hmda-analytics/src/main/scala/hmda/analytics/HmdaAnalyticsApp.scala

```diff
@@ -392,85 +379,85 @@ object HmdaAnalyticsApp extends App with TransmittalSheetComponent with LarCompo
         .mapAsync(1) { lar =>
           for {
             insertorupdate <- submissionId.period match {
```

Comment:
Could we replace this very long match statement with a map? And maybe do the same for the other match statements here? It would cut down on a ton of lines of code

Reply:
Hmm could you clarify a bit what you were thinking?

Reply:
Never mind, map isn't the right thing here. Not sure what I was thinking yesterday. 

What I meant is we can simplify this to 2-3 cases: `case Period(year, None) =>` and `case Period(year, Some(quarter)) =>` (and keep the final `case _ =>` case). Using a variable for the year and quarter will get rid of a ton of these match cases.

Reply:
Oh yeah let's do that, will take some time to update